plasmapy/analysis/fit_functions.py

```diff
@@ -0,0 +1,1141 @@
+                    \\right]
+
+        Parameters
+        ----------
+        x: array_like
+            Independent variables to be passed to the fit function.
+
+        x_err: array_like, optional
+            Errors associated with the independent variables `x`.  Must be of
+            size one or equal to the size of `x`.
+
+        rety: bool
+            Set `True` to also return the associated dependent variables
+            :math:`y = f(x)`.
+
+        Returns
+        -------
+        err: `numpy.ndarray`
+            The calculated uncertainties :math:`\\delta y` of the dependent
+            variables (:math:`y = f(x)`) of the independent variables `x`.
+
+        y: `numpy.ndarray`, optional
+            (if `rety = True`) The associated dependent variables
+            :math:`y = f(x)`.
+        """
+        x = self._check_x(x)
+        if x_err is not None:
+            x_err = self._check_x(x_err)
+
+            if x_err.shape == ():
+                pass
+            elif x_err.shape != x.shape:
+                raise ValueError(
+                    f"x_err shape {x_err.shape} must be equal the shape of "
+                    f"x {x.shape}."
+                )
+
+        a, alpha, m, b = self.params
+
+        exp_y, exp_err = self._exponential(x, x_err=x_err, reterr=True)
+        lin_y, lin_err = self._linear(x, x_err=x_err, reterr=True)
+        err = exp_err ** 2 + lin_err ** 2
+
+        if x_err is not None:
+            blend_err = 2 * a * alpha * m * np.exp(alpha * x) * (x_err ** 2)
+            err += blend_err
+        err = np.sqrt(err)
+
+        if rety:
+            return err, exp_y + lin_y
+
+        return err
+
+
+class ExponentialPlusOffset(AbstractFitFunction):
+    """
+    A sub-class of `AbstractFitFunction` to represent an exponential with a DC
+    offset.
+
+    .. math::
+
+        y =& f(x) = A \\, e^{\\alpha \\, x} + m \\, x + b\\\\
+        (\\delta y)^2 =&
+            \\left( A e^{\\alpha x}\\right)^2 \\left[
+                \\left( \\frac{\\delta A}{A} \\right)^2
+                + (x \\, \\delta \\alpha)^2
+                + (\\alpha \\, \\delta x)^2
+            \\right]
+            + (\\delta b)^2
+
+
+    where :math:`A`, :math:`\\alpha`, and :math:`b` are the real constants to
+    be fitted and :math:`x` is the independent variable.  :math:`\\delta A`,
+    :math:`\\delta \\alpha`, :math:`\\delta b`, and :math:`\\delta x` are the
+    respective uncertainties for :math:`A`, :math:`\\alpha`, and :math:`b`, and
+    :math:`x`.
+
+    """
+
+    _param_names = ("a", "alpha", "b")
+
+    def __init__(
+        self, params: Tuple[float, ...] = None, param_errors: Tuple[float, ...] = None,
+    ):
+        self._explin = ExponentialPlusLinear()
+        super().__init__(params=params, param_errors=param_errors)
+
+    def __str__(self):
+        return f"f(x) = A exp(alpha x) + b"
+
+    @property
+    def latex_str(self) -> str:
+        return fr"A \, \exp(\alpha \, x) + b"
```

Comment:
```suggestion
        return fr"A \exp(\alpha x) + b"
```
Would be helpful to avoid the extra formatting.